test/test_validation.js

```diff
@@ -58,3 +58,14 @@ describe('sprintfjs', function() {
         })
     })
 })
+
+describe('sprintfjs-prototype', function() {
+
+    function Klass() {}
+    Klass.prototype = {
+        get x() { return 2 }
```

Comment:
Test properties/getters which might have a value of `undefined`, test undefined properties (which should throw)